File: Algoritmos Traduzidos/Perl/Algoritmos_Filipe_Dias/105.pl
print "Digite sigla:\n";
$sigla = <STDIN>;
chomp ($sigla);
if ($sigla eq "RJ" || $sigla eq "rj")
{
	print "Carioca\n";
}
elsif ($sigla eq "SP" || $sigla eq "sp")
{
	print "Paulista\n";
}
elsif ($sigla eq "MG" || $sigla eq "mg")
{
	print "Mineiro\n";
}
else
{
	print "Outros estados\n";
}

File: Algoritmos Traduzidos/Ruby/Algoritmos_Filipe_Dias/105.rb
puts "Digite sigla:"
sigla = gets.chomp
if sigla == 'RJ' || sigla == 'rj'
  puts "Carioca"
elsif sigla == 'SP' || sigla == 'sp'
  puts "Paulista"
elsif sigla == 'MG' || sigla == 'mg'
  puts "Mineiro"
else
  puts "Outros estados"
end